static/js/tabs.js

```diff
@@ -0,0 +1,44 @@
+function initTabs() {
+  var tabLinks = document.querySelectorAll("a[href^='#']");
+
+  // prevent the page jumping around when switching tabs,
+  // whilst still using :target
+  // https://gist.github.com/pimterry/260841c2104f27cadc954a29b9873b96#file-disable-link-jump-with-workaround-js
+  [].forEach.call(tabLinks, function(link) {
```

Comment:
a `NodeList` now has a `forEach` method, so using this trick is no longer necessary.